src/pluggy/manager.py

```diff
@@ -135,6 +135,7 @@ def parse_hookimpl_opts(self, plugin, name):
             res = getattr(method, self.project_name + "_impl", None)
         except Exception:
             res = {}
+            normalize_hookimpl_opts(res)
```

Comment:
Hmm maybe we could do the same single conditional block like at line `111` at the end of this method?

Reply:
I can't figure out what "same single conditional block like at line 111" refers to, can you explain more?

Reply:
Ahh sorry my local version of the code is out of date. So basically a block [like this](https://github.com/pytest-dev/pluggy/blob/master/src/pluggy/manager.py#L115) at the end of this method which does the normalization if `res != None`. This might also mean we can remove line `116`? A brief look shows this is the only place that method is used internally so I think we might be ok with that change.

@RonnyPfannschmidt @nicoddemus do we use that function in `pytest` for some reason?

Reply:
[Doesn't look like it](https://github.com/pytest-dev/pytest/search?q=normalize_hookimpl_opts&unscoped_q=normalize_hookimpl_opts). @bluetech I think you should be fine to factor that call into `parse_hookimpl_opts()` and remove it from `116`.

Reply:
There is a test `test_parse_hookimpl_override` which overrides `parse_hookimpl_opts`, returns `{}` and expects it to work. I did not consider this option so the premise of my commit was wrong. I removed it.